From c412a2e8017bf7e6cca107aaad91905c6b85a634 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Thu, 13 Apr 2017 10:29:28 -0400
Subject: [PATCH 1193/2094] drm/dp-helper: DP_TEST_MISC1 should be
 DP_TEST_MISC0

Bring this in line with spec and what commit in upstream drm tree.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
---
 include/drm/drm_dp_helper.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/drm/drm_dp_helper.h b/include/drm/drm_dp_helper.h
index 4b14a76..d6a5015 100644
--- a/include/drm/drm_dp_helper.h
+++ b/include/drm/drm_dp_helper.h
@@ -419,7 +419,7 @@
 
 #define DP_TEST_PATTERN			    0x221
 
-#define DP_TEST_MISC1                       0x232
+#define DP_TEST_MISC0                       0x232
 
 #define DP_TEST_CRC_R_CR		    0x240
 #define DP_TEST_CRC_G_Y			    0x242
-- 
2.7.4

